maintainers/working_groups/learning_journey/tutorial-outlines/sharing-dependencies.md

```diff
@@ -0,0 +1,64 @@
+# Sharing dependencies between `default.nix` and `shell.nix`
+
+## Assumptions
+- The reader has seen a derivation before.
```

Comment:
We may want to specify which tutorials or explanations should have been read first. Is it "packaging existing software"?